FlashEnglish/Sources/FlashEnglishApp.swift

```diff
@@ -37,6 +42,11 @@ final class AppDelegate: NSObject, UIApplicationDelegate, UNUserNotificationCent
         }
 
         return true
+
+
+
+
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/26f8fd3d00e8b28f320403772b345ccefdcdf0b2/FlashEnglish/Sources/FlashEnglishApp.swift#L49"></span>Limit vertical whitespace to a single empty line; currently 5<br />
<code>vertical_whitespace</code> <code>FlashEnglishApp.swift:49</code></td>
    </tr>
  </tbody>
</table>